canonical/ubuntu.com
# add redirect

## Done

- Redirect Ubuntu Kios

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Try to visit /tutorials/secure-ubuntu-kiosk
- Should redirect to /docs/make-a-secure-ubuntu-web-kiosk

## Issue / Card

Fixes [#WD-17299](https://warthogs.atlassian.net/browse/WD-17299)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
